cfy_manager/utils/certificates.py

```diff
@@ -441,14 +441,26 @@ def use_supplied_certificates(component_name,
 
     Returns True if supplied certificates were used.
     """
-    cert_src, key_src, ca_src, key_pass = check_certificates(
-        component_name,
-        cert_path=cert_prefix + 'path',
-        key_path=key_prefix + 'path',
-        ca_path=ca_prefix + 'path',
-        key_password=key_password_prefix + 'password',
-        require_non_ca_certs=False,
-    )
+    key_entry = key_prefix + 'path'
+    ca_entry = ca_prefix + 'path'
+    cert_entry = cert_prefix + 'path'
+    password_entry = key_password_prefix + 'password'
+
+    if key_destination and not cert_destination and not ca_destination:
+        # If there's only a key, our check function won't work, so skip it
```

Comment:
maybe add a warning message for future debugging sake

Reply:
Added